src/contexts/DataContext.js

```diff
@@ -0,0 +1,42 @@
+import { createContext, useState } from 'react';
+import { DateTime } from 'luxon';
+
+export const DataContext = createContext();
+
+const date = DateTime.local(); 
```

Comment:
Can we make this a method instead and call it createDate?  It's fairly dangerous to modify date inside a method with a different scope.  What if something else ends up modifying the date?  Since we're passing it along to many other functions.  That will also modify this date.